_episodes/l1-03-intermediate_concepts.md

```diff
@@ -267,6 +295,19 @@ $ git revert --no-edit COMMIT_HASH
 This is yet another good example of why making separate commits for each change is a
 good idea, so they can, potentially, be reversed if needed in the future with no fuss.
 
+> ## `reset` vs `reverse`
```

Comment:
Ha, I didn't know `reverse` was a thing!

Reply:
I don't think `reverse` is a command. 

```suggestion
> ## `reset` vs `revert`
```